Scripts/script-CheckSender.yml

```diff
@@ -1,20 +1,31 @@
 commonfields:
-  id: CheckSender
+  id: CheckSenderPy
   version: 1
-name: CheckSender
+name: CheckSenderPy
 script: |-
-  var fromRE = /<b>From:.*href="mailto:(.*)"/ig;
-  var from = fromRE.exec(incidents[0].details);
-  var res = [];
-  var rep = executeCommand('pipl-search', {email: from[1]});
-  Array.prototype.push.apply(res, rep);
-  res.push({Contents: 'no', Type: entryTypes.note, ContentsFormat: formats.text});
-  return res;
-type: javascript
+  import re
+  email = ''
+  if 'email' in demisto.args():
+      email = demisto.args()['email']
+  else:
+      sender = re.search(r".*From\w*:.*\b([A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,})\b", demisto.incidents()[0]['details'], re.I)
```

Comment:
I'm not liking it. I think we should just receive a parameter of where to look and then pass the relevant context parameter.Let's not hard-code the demisto.incidents()[0]['details'] here. So, 2 parameters - email and body which you'll regex the email from.